packages/opencode/script/ppt-quality-eval/route-skills/svg-pptx-native/scripts/svg_finalize/flatten_tspan.py

```
@@ -0,0 +1,818 @@
+
+    return ne
+
+
+def process_svg_file(
+    src_path: str,
+    dst_path: str,
+    merge_paragraphs: bool = False,
+) -> bool:
+    """Flatten eligible tspan lines in one SVG file."""
+    try:
+        tree = ET.parse(src_path)
```

Ruff (0.15.20)

[error] Using `xml` to parse untrusted data is known to be vulnerable to XML attacks; use `defusedxml` equivalents

(S314)